docs/semgrep-app/integrations.md

```diff
@@ -1,19 +1,163 @@
+6. After you have enabled **Incoming Webhooks**, follow one of these steps:
+    - Generate a **Webhook URL**.
+    - Request to add a new webhook from a workspace owner.
+7. After you've obtained the Webhook URL, return to Semgrep and create a new Slack integration. Name the integration and copy-paste the URL into the **Slack URL** field. You may opt to select the **Inventory** flag to include notifications about Code Asset Inventory findings.
+8. You can **Test** then select which channel you'd like Semgrep to start posting to.
+![Screenshot of the Slack Request Webhook  ](../img/slack-integration-hooktest.png)<br />
+9. After you've verified that everything is working, remember to **Save** the final configuration.
+10. Remember to turn notifications on by going to the **Rule board**, clicking on the **gear icon,** then clicking on the toggle.
+
+Here is a sample of a Slack message sent from Semgrep with findings:
+
+![Screenshot of Slack message with findings ](../img/slack-semgrep-findings.png)<br />
+
+#### Additional resources
+
+* https://api.slack.com/apps
+* https://api.slack.com/messaging/webhooks#enable_webhooks
+
+#### See also
+
+[Notifications -> Slack](notifications.md/#slack)
+
+### Email
+
+Email integration allows Semgrep to send new findings to an email address of your choice.
+
+To set up email integration:
+
+1. In **Integrations,** click **Add Integration.**
+2. Click on **Email**.
+3. Enter a **Name** for the integration.
+4. Enter the **Email address** that will receive Semgrep findings.
+5. Select the **Inventory** check box if you would like to receive notifications about Code Asset Inventory findings.
+6. Click **Save.**
+7. Remember to turn notifications on by going to the **Rule board**, clicking on the **gear icon,** then clicking on the toggle.
+
+Here is a sample of an email sent from Semgrep with findings:
+
+![Screenshot of Semgrep email with findings ](../img/integrations-email-findings.png)<br />
+
+#### See also
+[Notifcations -> Email](notifications.md/#email)
+
+### Jira
+
+Jira integration is a feature available in Semgrep's Team tier and above.
+
+This integration allows you to create Jira tickets directly from the **Findings** page with relevant information about a particular finding.
+
+To set up Jira integration:
+
+1. In **Integrations,** click **Add Integration**.
+2. Click on **Jira.**
+3. Enter a **Name** for the integration.
+4. Enter the **email address** used for the Atlassian account.
+5. Enter your Atlassian **domain URL**.
+6. Enter your **Project key**. This is the prefix for tasks created within a project. Semgrep will create issues to the project identified here.
+7. Enter the **Issue type.** This is the type of issue for Semgrep findings, for example, *Bug.*
+8. Enter the **API Token**. Tokens are generated through this link: [Manage API Tokens](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account).
```

Comment:
```suggestion
8. Enter the **API Token**. Tokens are generated through the following link: [Manage API Tokens](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account).
```

Does a user need to click the link? If so, say it in imperative form: To generate the token, click the following link.
For more information, see [Google style guide](https://developers.google.com/style/link-text?hl=en). :-)